server.js

```diff
@@ -1,16 +1,29 @@
 require("dotenv").config();
+const path = require("path");
 const express = require("express");
-const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
```

Comment:
We don't need cors anymore, as our server and our client will be served from the same domain